builtin/bins/dkron-executor-kafka/kafka.go

```diff
@@ -65,7 +66,7 @@ func (s *Kafka) ExecuteImpl(args *dktypes.ExecuteRequest) ([]byte, error) {
 	config.Producer.Return.Successes = true
 	config.Producer.Return.Errors = true
 
-	brokers := []string{args.Config["brokerAddress"]}
+	brokers := strings.Split(args.Config["brokerAddress"], ",")
```

Comment:
What if people accidentally use an additional space after the comma? Does `NewSyncProducer` handle that properly or should it be taken into account here?

Reply:
What is your suggestion for that cases? I had that doubt to but I think failing to deliver a message because one of the many addresses passed is wrong it will be worst. This still applies to the single broker, if we provide an extra space it will have the same problem. 🤔 

Maybe pointing to [here](https://jaceklaskowski.gitbooks.io/apache-kafka/content/kafka-properties-bootstrap-servers.html) in the docs will be enough?